onnxscript/rewriter/llama_rule_sets.py

```diff
@@ -12,6 +12,26 @@
 import onnxscript.rewriter.pattern as orp
 
 
+class SqueezeReshape(orp.RewriteRuleClassBase):
+    """Replaces ``Reshape(Squeeze(x), [-1]])`` with ``Identity(x)`` for 1D x.
+
+    This pattern arises from the translation of pytorch symints.
+    """
+
+    def __init__(self):
+        super().__init__("SqueezeReshape1d", remove_nodes=False)
+
+    def pattern(self, op, x):
+        return op.Reshape(op.Squeeze(x), [-1])
```

Comment:
Does it work if reshape have multiple users?

Reply:
Yes. It is not different from any other rewrite-rule in that regard.